acceptance/tests/resource/mount/modify.rb

```diff
@@ -29,6 +29,9 @@
   step "(setup) create mount point"
   on(agent, "mkdir /#{name}", :acceptable_exit_codes => [0,1])
 
+  step "(setup) ensure loop kernel module is installed"
+  on(agent, "modprobe loop", :acceptable_exit_codes => (0..254))
```

Comment:
`mount` returns a respectable number of meaningful error codes; checking and displaying those might eas debugging in the future.